org.adempiere.base/src/org/compiere/wf/MWorkflow.java

```diff
@@ -125,46 +127,43 @@ public static MWorkflow getCopy(Properties ctx, int AD_Workflow_ID, String trxNa
 			, String trxName //Bug 1568766 Trx should be kept all along the road		
 	)
 	{
-		String key = "C" + AD_Client_ID + "T" + AD_Table_ID;
 		//	Reload
-		if (s_cacheDocValue.isReset())
+		Map<Integer, MWorkflow[]> cachedMap = s_cacheDocValue.get(AD_Client_ID);
+		if (cachedMap == null)
 		{
-			final String whereClause = "WorkflowType=? AND IsValid=?";
+			final String whereClause = "WorkflowType=? AND IsValid=? AND AD_Client_ID=?";
 			List<MWorkflow> workflows;
-			try {
-				PO.setCrossTenantSafe();
-				workflows = new Query(ctx, Table_Name, whereClause, trxName)
-						.setParameters(new Object[]{WORKFLOWTYPE_DocumentValue, true})
-						.setOnlyActiveRecords(true)
-						.setOrderBy("AD_Client_ID, AD_Table_ID")
-						.list();
-			} finally {
-				PO.clearCrossTenantSafe();
-			}
+			workflows = new Query(ctx, Table_Name, whereClause, trxName)

```

Comment:
tested successfully - creating business partner in two clients with workflow DocValue same as GardenWorld